views/foo.handlebars

```diff
@@ -0,0 +1,25 @@
ymsRtsVVZ2vAiQp3+k02+1/jBpZApg78LUFnzws7RAiMCqLf3rGlVnyXgWhOaPBgGyTSLVIBrDZMUDs8ACumWs2SvT77eNqdGQiSP3APYanCpPI/KqGCAqw+DXaoi/zfSLwSAXoJHtAqg97GhqpAmiFV7Y4agh/vL0JwWlDWFC6i+AEkKgBWqLf3VD4zpYFXvrL+SNflllg1nLAU/UbLMY7QwW2dCQUgCW2NzL/1sllOkvbBvku5hcgKwXQA1ISBAqOYBnpAGJ5al18LCQpb9ke0p3wfYaECibLsRYJmIZlO7QQPBsnzoqxqeksnrJBsAFyHYBVAUmG0qJ4MBo5QGjh3db9VWmf15MvzU0uYDQbgG0iDC3lmIKecR2yEj0YOvao2dZ+ksbvVhCJGiwZRSkHeLLvVVX2Io8keoS1t5relwdOvAFYIu6Fr5jJUHaIdZcAcQaEBUDMD+2u0bq9P8q5c9NkLgAdlGEegg9VwCRPy0sxoSCbe7SE3lUTX9LtMqizthIUKY+arvwAv9Tqr5gkyz7rQaH8vQXFIU5WBu6uABWEqQa0LwULfCn3hf9j6X6fkj0S40Pgudhfr0BBGpYss2CBf5XbG8pNz9Du8BcVl6r3pahIYqzCiO0U4hbgy/wp0UhMD+W6isrb2jVormij5YkqFoFNAcwpLpKcCyZHzvv+1NS3pI9NQlmCapMxTItKd2Egv+i6gtDurRRX9nzlgSeyryxqV7XGYB+ivs15t5ywswP+Sub4Fkx/aFejizA0ugEQvORAM7/XX3OUZfHb3fU/LStzY9c8rFsTCO4YNolPaHcCOm3SNy1ExY/H7F9sdzM2kQPC9n0W3c25rg2qUDFACKtJjYOefzPkhdiwm3MjyI9Ya3uanaIcm0/AP0Y6LzD8q97an5I+nFTbX4+FNtT75jNkB5iHUgDGJUNQL9F6L7DPlJTArWfi0rzM5VtT/e1Fv2tvRhQCzGyHYB+jPgqZh8pG4ioK6OfgSI9xr3AUsy8ugPQ+d/weIXWTkCjq0pvkVTt3456hzOjRQMoDKYzTcNWrKrFzT8csa6Y/t5cmv7OToct+QCkpxWuQKZN0kWf6i/v6yDRWwV9JWzrVCQtRZjQ68sGsicpGECg7fe+MP5y7arilK8SNyR1dZdh2KMDUMyo0RNrAXTribbH0qYljN7K902VnKVbx3GpOGD/k/hIcyih81+ykIhG/2A+y6NP2fTEtc81SXhh2xpyLGQIp9U9DBGsdE7MZ2np9qe01PYnOgpww65slaNRfQBl16jQ7AXMZ5n5l2P+uL7hVzHp0dWWBqDb0bJrbKhMQPJbEj3L9OsfijPh0ihDI1UJ9D1ZDlb8qai9yfSTz6iugNmULYH4/Lk8gLJrkPB4ANZvCnPRZ0n2OzsbTg1PqcmSylm9nkBy0OiZaGOR+5KDnrCz87FiA8YHfZMeizJUdg0Y3TkljQNjs/mcSnY/jD9/OYCEZa+vB0ThnKtx2TVsGL5BPcWcvcgRc7L5+gtXlr0URjAQFZm5Agv+UI8w7TxGH2KQFXZui5/1mRF84B3YwghCaoiy4msI6dYj5POG7Gvx7Av0k/kXyw7H7DoNcAS8T2VORlByDeSY8ifmX6s9R72NNPk7xpuWuOjRRWDWFNag+B4kyh/uplP5D56F01dh5/7br+OavSzSLFJGsB4UX6M7oelXy8BfnPwwOf420ZGH8HQeBNsxcH8Q+oeFJQ5Mv7D+LYWfU2Hyw2T02ftI6mB5eZ4FQbQmrQ7+cc9L3wpCOMr/UOEfLf4cC+z9yu2JL8ds9XKwCNvt+ZSocLq4MC0C5Q99b8z+RH+eQ0o/bB/u7RrD4d1s9d7brkXmRYFpESbSgUHqvwZ/Rpz985fcJvQpzB7fe1EQpHohhfIHw0gvph6s8axQnIR/U+5LMXu81v0w8sdyuhC/bb3I+r/fZz9qXh3xt0H5o23kMVMUeC9NuD2yApQ/UqUnNoPgoBlXj1YB+XdRR0n+mKXTp++5D+/rQe0nRgbX2VZurhtxa6odaPkTf7zsn68arrEygD/rXlg6oLISkP/nizl7AvL/bC1tb5iY2nEdAvKvupegsYD8K/66etRfBvK3vJumefhH+H9vXeEbAfy7X1nQ/KuA+lW3MUlKXQD/8t7ZJgP5uxHsm/Bv8P87pbXvwD/Bv2YPUpPwf/77Rdtx+e86zj92nL/vOH9SKHfY/5LNXXfjN+j+d52/6UIUR0C2SN3Nv1qktcTd/Bf6x+Lvacr4GyDt87YHWBoI0n/obv0TDp/and9qJEj/qrv1f+RfeXq0sTBcB+EUoH+73hmiJuEX3Ofiyn6vBrguv/jqiaYDLh9wN4GHq3/cTWDg/Iu7ATQcP3U3AIXzU+4GoMb7pBwC3P7gbgAHDbg29zc0E7CB5HAABw3Q7gZAcP6x62wABP1jljdQNBBwOsTdAAJ6/N0NIOBol7sdKHCA090OJri9y90A4lZ4S7GLwATeWQeMB+CddcAPjjtgvP/H2Qz+TjpB4h7w+jpnt5DwBKq7DhgTAGdboPEAv7MODAJodzNgekjWVQeGF3g5W0LHG1yczSB/SS8ddA80AHI1g8QAyFkHhkf4K98t2ljgawedLeFuzO9UcQb0/m1XHRgaUGcdGF5hZ34tngPAt78468Dwle/VV4A3FfeOl0DxnTzONsGBApe9f6PZwD08Zx0YvlPI2RIoeuDKt9s3FpACOOvAMIR2/RSGu6doYRPM3SYg6IJzdw+vddpxeg8PFMDdEigogLslUNoG6moGiR7AXQcMVXR3HTCEQM7uQbYgBHI3A4aLgJwt4ebY5gBddwMIogDudhG3iAK4m8G3iAK4HADlHsDdEnqOs9DlAC5/EZXLAVzeiBKO3A3gtvDjxOEArtW6Ojlxev5fsrIXTTQfy1R/U5tLGGaGNyW5hIPIbQFapdrLyt3CNPvYN4VPYdYvedOHC1h+9dXP/wOWwyhjDKSUwgAAAABJRU5ErkJggg==">
```

Comment:
Is this an entire image base64 encoded? We should create a static directory for static content